shared/components/myPage.js

```diff
@@ -211,6 +212,31 @@ class MyPage extends Component {
     );
   }
 
+  renderTextBoxInUserBox() {
+    return (
+      <View style={{flexDirection: 'column', flex: 225}}>
+        <View style={{flex: 35}}>
+          <Text style={[styles.myPageTextUserName, styles.fontRobotoMedium]}>
+            {(
+            this.props.userName !== '' &&
```

Comment:
Can you change this more simple? I think there are too many conditions. are there all necessary?